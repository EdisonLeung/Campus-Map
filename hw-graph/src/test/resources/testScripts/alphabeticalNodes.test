# Test whether it will print nodes in alphabetical order after adding 3 nodes in
# reverse alphabetical order

# Create graph
CreateGraph alphabeticalOrder

# Add nodes in reverse alphabetic order
AddNode alphabeticalOrder c
AddNode alphabeticalOrder b
AddNode alphabeticalOrder a

# Prints the nodes
ListNodes alphabeticalOrder