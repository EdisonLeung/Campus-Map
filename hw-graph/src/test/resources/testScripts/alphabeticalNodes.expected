# Test whether it will print nodes in alphabetical order after adding 3 nodes in
# reverse alphabetical order

# Create graph
created graph alphabeticalOrder

# Add nodes in reverse alphabetic order
added node c to alphabeticalOrder
added node b to alphabeticalOrder
added node a to alphabeticalOrder

# Prints the nodes
alphabeticalOrder contains: a b c